framework/src/modules/http_api/controllers/transactions.js

```diff
@@ -45,15 +45,16 @@ function TransactionsController(scope) {
 }
 
 function transactionFormatter(transaction) {
-	const result = _.omit(transaction, ['requesterPublicKey']);
+	const result = _.omit(transaction, [
+		'requesterPublicKey',
+		'recipientPublicKey',
+	]);
```

Comment:
Can't these properties be removed from Storage all together?

Reply:
Removed and added migration to drop `requesterPublicKey`